src/WebUI/Common/KubeSummary.scss

```diff
@@ -7,11 +7,16 @@
     word-wrap: normal;
 }
 
+#main-content {
```

Comment:
this is very bad, if someone has an id of main-content then this going to affect them. we should use a css or parent css class, so this is not affected others.

Reply:
Removed css from id main content, as the height should be set by the parent that is hosting the kubernetes container